### PR TITLE
tomb: add zsh dependency

### PR DESCRIPTION
**Pull request by mamapitufo**

### CODE REVIEW

File: srcpkgs/tomb/template [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+revision=2
 wrksrc=Tomb-${version}
 build_style=gnu-makefile
 hostmakedepends="pkg-config"
 makedepends="libnotify-devel gtk+-devel"
-depends="sudo gnupg cryptsetup pinentry"
+depends="sudo gnupg cryptsetup pinentry zsh"
 short_desc="A minimalistic commandline tool to manage encrypted volumes"
 maintainer="Diogo Leal <diogo@diogoleal.com>"
 license="GPL-3"
```
Comment (Hoshpak) [inline srcpkgs/tomb/template]: Please update to use SPDX identifier. According to the license in the file header `GPL-3.0-or-later` would be the right one.

Comment (mamapitufo) [reply srcpkgs/tomb/template]: Should I add a note on the commit message about the license field change then?

Comment (Hoshpak) [reply srcpkgs/tomb/template]: that's not necessary. It's just an update regularly applied to all templates that don't use SPDX identifiers yet.

Comment (mamapitufo) [reply srcpkgs/tomb/template]: ok, thanks. I'll keep it in mind if I make any other changes in the future.

**Pull request merged**